apps/editors/helpers.py

```diff
@@ -799,7 +799,7 @@ class ReviewFiles(ReviewBase):
 
     def set_data(self, data):
         self.data = data
-        self.files = data.get('addon_files', None)
+        self.files = self.version.files.all()
```

Comment:
Actually, I removed this modification entirely, there was no reason to change anything here